autobuild/hash_algorithms.py

```diff
@@ -64,3 +64,7 @@ def verify_hash(hash_algorithm, pathname, hash):
 @hash_algorithm("md5")
 def _verify_md5(pathname, hash):
     return common.compute_md5(pathname) == hash
+
+@hash_algorithm("blake2b")
+def _verify_blake2b(pathname, hash):
+    return common.compute_blake2b(pathname) == hash
```

Comment:
Nit: newline at end of file needed. You could autofix these with the repository's [pre-commit](pre-commit.com/) configuration.